prestodb/transaction.py

```diff
@@ -90,26 +91,18 @@ def begin(self):
 
     def commit(self):
         self._request.transaction_id = self._id
-        response = self._request.post(COMMIT)
-        if not response.ok:
+        query = prestodb.client.PrestoQuery(self._request, COMMIT)
+        try:
+            return list(query.execute())
```

Comment:
What would be the output of this? Hopefully it's a one-line successful info. It might be good to do some assertion with the output to tell if it succeeded or not. Ideally, this function returns nothing or a boolean to indicate the result.

Reply:
Returning a value here is misleading as this is a *side-effects* only method: either it worked or it raises an exception. The purpose of the exception is to know if the caller should retry the `COMMIT` or *rollback* the transaction.

I'd prefer to remove the `return`, which is equivalent to a `void` type in Python though it would return `None` (I didn't find how to express this with [Mypy](http://mypy.readthedocs.io/)).